official/v7/entity_response.go

```diff
@@ -133,7 +141,7 @@ func checkErrorFromResponse(response *esapi.Response) error {
 
 	errorBody, ok := responseBody["error"].(map[string]interface{})
 	if !ok {
-		return errors.New("failed to decode error from response")
+		return errors.New("failed to decode error from response: " + responseBody["error"].(string))
```

Comment:
This approach to error messages creates a variable error message. A constant error message is easier to search for in the logs and can be achieved with the errors.E() method:

```
return errors.E("failed to decode error from response", errors.KV("error", responseBody["error"]))
```

Reply:
It looks like this package is not using foundationkit errors, if that's the case, please disregard the comment above.